docs/Installation/Installation_Guide.rst

```diff
@@ -172,6 +172,54 @@ configuring persistent voloumes at 500 Gi/node.
 
    It is a best practice to install Zenko on a fresh cluster.
 
+Pod Disruption Budgets
+**********************
+
+Zenko relies on several stateful services that require a minimum number
+of pods to function with high availability, with resilence under many
+outage scenarios. Pod disruption budgets set how many pods of a given
+application can safely fail and continue to operate normally before
+Kubernetes disables access to the service.
+
+  .. note::
+
+     If you are installing Zenko in a high-availability production environment,
+     set a pod disruption budget. If you are installing a basic Zenko deployment
+     (for testing or familiarization, for instance) you can skip this step.
+
+The following installed applications allow for configuring disruption
+budgets:
+
+  - MongoDB
+  - Redis
+  - Zenko-Quorum
+
+In a three-node cluster, Zenko configures a ``maxUnavailable`` budget
+of 1 by default. However in larger clusters, this should be increased to
+match the level of high availablity required. For example, a five-node
+cluster can have up to two ``maxUnavailable`` pods per application.
+
+In addition to configuring the node count at install time, you should
+also configure the pod disruption budgets. This is an example for
+configuring the budgets for a 5 node installation.
+
+.. code-block:: yaml
```

Comment:
@wabernatScality Does this formatting of this section look correct to you? In the Github preview it looks like it might be indented but I'm not sure if that is just a result of how it renders on Github.